src/functionalTest/java/uk/gov/hmcts/reform/amapi/functional/FunctionalTestSuite.java

```diff
@@ -30,16 +35,17 @@
 import java.time.Instant;
 import java.util.List;
 import java.util.UUID;
-import javax.sql.DataSource;
 
 import static java.lang.System.getenv;
 import static uk.gov.hmcts.reform.amlib.enums.AccessorType.USER;
 import static uk.gov.hmcts.reform.amlib.enums.Permission.READ;
+import static uk.gov.hmcts.reform.amlib.enums.SecurityClassification.PUBLIC;
 
 @TestPropertySource("classpath:application-functional.yaml")
 @Slf4j
 @ActiveProfiles("functional")
-@SuppressWarnings({"PMD.DataflowAnomalyAnalysis", "PMD.ConfusingTernary", "PMD.JUnit4TestShouldUseTestAnnotation"})
```

Comment:
too many warnings are suppressed, we should avoid and use this annotation only when utmost necessary

Reply:
Resolved  "PMD.ConfusingTernary". The others are required, otherwise too many changes in the code.